test/testinput/dirac.yml

```diff
@@ -0,0 +1,24 @@
+background error:
+  covariance model: umdsstCovar
+  horizontal_length_scale: 2.2e6
+  maximum_condition_number: 1.0e6
+  standard_deviation: 1.8e7
+dirac:
+  date: 2010-01-01T12:00:00Z
+  ixdir: [20, 50, 100, 150, 200, 250, 300, 350]
+  iydir: [10, 25, 50,  75,  100, 125, 150, 175]
+geometry:
```

Comment:
`dirac.yml` needs to be added to the relevant section of the `test/CMakeLists.txt` file https://github.com/UMD-AOSC/UMD-SST/blob/40244f0b0a95e66bd086b9269fe980e161f30cc5/test/CMakeLists.txt#L1

It is not being linked into the build/test directory and so the ctests are currently failing

Reply:
... same thing with the `testref/dirac.ref` file

Reply:
Done.